helm/hpcc/templates/dali.yaml

```diff
@@ -24,6 +24,14 @@ spec:
       serviceAccountName: "hpcc-default"
       initContainers: 
         {{- include "hpcc.checkDaliMount" (dict "root" $) | indent 6 }}
+        {{- if not ( $sashaServices.disabled ) -}}
+        {{- range $sashaName, $_sasha := $dali.services -}}
+        {{- $sasha := ($_sasha | default dict) -}}
+        {{- if and (not $sasha.disabled ) ( $sasha.storage) }}
```

Comment:
can perhaps ignore edge case for now, and open a separate JIRA to fix it later,
but if you had 2 sasha services using same storage, this template code would generate 2 init containers, not sure if it would fail or happen to work, but should really only generate 1 per unique storage.plane

Reply:
Not sure what to do.  Maybe one of 
1) remove this part of code and open a separate JIRA
2) break out the loop when one sasha.storage found and handled

Reply:
I see your following review. I will keep the code for now and open a new JIRA

Reply:
New JIRA: https://track.hpccsystems.com/browse/HPCC-25798

Reply:
@xwang2713 - that looks like an unrelated JIRA ("aws-cpp-sdk-core build fails on Ubuntu 21.04")
can you reference correct one and link it to this JIRA (HPCC-25685)